umbraco/Umbraco-CMS
# Allow deleting files in upload property editor

### Description
When you remove an image from a Upload File property editor, it still sends data, with `src=""`. The backend expected a null value. Not backend respects both and delete the existing data when  `src=""` is send

### Tests
- Setup doc type with an `Upload file`
- Create a document with an uploaded image
- Try to remove that image and save. - It should both be deleted from the document and removed from disk
